cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,41 @@
-describe('', () => {
+const { faker } = require('@faker-js/faker');
+
+describe('Article CRUD', () => {
   before(() => {
+  });
+  const articleTitle = faker.word.words(1);
+  const articleTopic = faker.word.words(2);
+  const articleDescription = faker.word.words(5);
+  const articleTag = faker.word.words(1);
 
+  it('should create new article', () => {
+    cy.task('generateUser').then((user) => {
+      cy.login(user.email, user.username, user.password);
+    });
+    cy.contains('.nav-link', 'New Article').click();
+    cy.get('[placeholder="Article Title"]').type(articleTitle);
+    cy.get('[placeholder="What\'s this article about?"]').type(articleTopic);
+    cy.get('[placeholder="Write your article (in markdown)"]')
+      .type(articleDescription);
+    cy.get('[placeholder="Enter tags"]').type(`${articleTag}{enter}`);
+    cy.contains('.btn', 'Publish Article').click();
+    cy.url().should('contain', 'article/');
   });
 
-  it('', () => {
+  it('should delete the article', () => {
+    cy.task('generateUser').then((user) => {
+      cy.login(user.email, user.username, user.password);
+    });
 
-  });
+    cy.createArticle(articleTitle, articleTopic, articleDescription)
+      .then((response) => {
+        const slug = response.body.article.slug;
+
+        cy.visit(`article/${slug}`);
+      });
 
-  it('', () => {
+    cy.contains('.btn', 'Delete Article').click();
 
+    cy.contains('.nav-link', 'Global Feed').should('be.visible');
   });
```

Comment:
In this test case, you are creating an article and then deleting it. It might be better to separate these into two different test cases: one for creating an article and another for deleting it. This way, if the creation test fails, it won't affect the deletion test and vice versa.